src/components/uxiconbaritem/uxiconbaritem.hbs

```diff
@@ -0,0 +1,4 @@
+<a class="item">
+    <img src="{{src}}">
+    <label>{{label}}</label>
```

Comment:
Are you enforcing passing a label to control input? Rather than just passing it through `{{yield}}`?